### PR TITLE
Refactored object selectors, corrected permission names.

### PR DESCRIPTION
**Pull request by eberonsh**

### CODE REVIEW

File: shuup/admin/modules/permission_groups/__init__.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_model_url(self, object, kind, shop=None):
         return derive_model_url(PermissionGroup, "shuup_admin:permission_group", object, kind)
 
     def get_extra_permissions(self) -> Iterable[str]:
-        return ["permission_group.object_selector"]
+        return ["group.object_selector"]
```
Comment (chessbr) [inline shuup/admin/modules/permission_groups/__init__.py:43]: is this correct?

**Review by chessbr — CHANGES_REQUESTED** [review]
I think you should create an utils that will return the permission for a model this way we can have consistent results across the platform

File: shuup/admin/modules/permission_groups/__init__.py (hunk 1 above)
Comment (eberonsh) [reply shuup/admin/modules/permission_groups/__init__.py:43]: Yes, PermissionGroup comes from: 
`from django.contrib.auth.models import Group as PermissionGroup`

Comment (chessbr) [reply shuup/admin/modules/permission_groups/__init__.py:43]: ah, makes sense

**Review by chessbr — APPROVED** [review]

File: shuup/admin/modules/services/__init__.py [hunk 2]
```diff
@@ -55,12 +56,12 @@ def get_model_url(self, object, kind, shop=None):
         return derive_model_url(self.model, self.url_name_prefix, object, kind)
 
     def get_extra_permissions(self) -> Iterable[str]:
-        return ["carrier.object_selector", "paymentmethod.object_selector"]
+        return [get_object_selector_permission_name(Carrier), get_object_selector_permission_name(PaymentMethod)]
```
Comment (eberonsh) [inline shuup/admin/modules/services/__init__.py:59]: It seems I missed the permission for ShippingMethod.

**Review by chessbr — APPROVED** [review]

**Pull request merged**